src/api/ans.ts

```diff
@@ -33,17 +44,96 @@ export class ANS {
   }
 
   /**
-   * Registers a new domain or subdomain name
+   * Retrieve the expiration time of a domain name or subdomain name.
+   *
+   * @param args.name - A string of the name to retrieve
+   *
+   * @returns number as a unix timestamp in seconds.
+   */
+  async getExpiration(args: { name: string }): ReturnType<typeof getExpiration> {
```

Comment:
curios what are the benefits for `ReturnType<typeof getExpiration>` - it is not a pattern we use in the sdk

Reply:
If your api function is just a light wrapper around the internal function, this makes it so you don't need to duplicate the types. These functions are simple enough were the inferance doesn't get us much, I can switch them over.